### PR TITLE
Proofreading and minor pre-release fixes

### PR DESCRIPTION
**Pull request by daniel-orlov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,39 +7,37 @@
 
 ## Purpose
 
-$${\textsf{\color{#1ab458}Discussion closure should rest with the initiator.}}$$
```
Comment (daniel-orlov) [inline README.md:10]: LaTeX doesn't get rendered properly in Markdown, so I switched to HTML

File: README.md [hunk 2]
```diff
@@ -7,39 +7,37 @@
...
+| days-before-close | The number of days to wait before closing a stale discussion. This is a required field.                       |     Yes      |                                     |                       |
+| close-unanswered  | If set to `true`, stale discussions that have not been marked as answered will also be closed.                |      No      | `true`, `false`                     | `false`               |
+| category          | The category of discussions to close. If not specified, all categories will be considered.                    |      No      |                                     | All, no filtering     |
+| close-reason      | The reason to use when closing a discussion.                                                                  |      No      | `DUPLICATE`, `OUTDATED`, `RESOLVED` | `OUTDATED`            |
+| dry-run           | If set to `true`, the processor will run in debug mode without performing any operations on live discussions. |      No      | `true`, `false`                     | `false`               |
```
Comment (daniel-orlov) [inline README.md:26]: Some minor changes for increased clarity

File: README.md [hunk 3]
```diff
@@ -65,7 +63,7 @@ jobs:
 
     steps:
       - name: Run action
-        uses: steffen-karlsson/stale-discussions@v1
+        uses: steffen-karlsson/stalesweeper@v1
```
Comment (daniel-orlov) [inline README.md:66]: Fixed it here and everywhere else, otherwise it was still pointing to the old name

File: README.md (hunk 1 above)
Comment (steffen-karlsson) [reply README.md:10]: Not on phone but in browser it's better, in GitHub Markdown, style tags etc. are stripped for safety reasons, this coloring etc is not there. It's recommended to use LaTeX

File: action.yml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 name: 'StaleSweeper'
 description:
-  'Automatically closes discussions that have not had recent interaction'
+  'Automatically closes stale discussions to declutter your GitHub repository.'
```
Comment (steffen-karlsson) [inline action.yml:3]: Hmm, I guess not your GitHub repository per say? More like the discussion fora, but maybe it's fine

Comment (daniel-orlov) [reply action.yml:3]: Let's change it to something as clear as possible 👍🏻 

However, to be honest, never heard the word fora before Denmark, do you mean 'forum'? 🤔

File: README.md (hunk 1 above)
Comment (daniel-orlov) [reply README.md:10]: Okay, makes sense 👍🏻 
You are right - It didn't work on my phone, in GH app, but it didn't work in my IDE interpreter either.

But I am okay with LaTeX :)

Comment (steffen-karlsson) [reply README.md:10]: Works fine in WebStorm :)

File: action.yml (hunk 4 above)
Comment (steffen-karlsson) [reply action.yml:3]: Fora is forum in plural 😅

Comment (daniel-orlov) [reply action.yml:3]: Dejligt 😅

**Review by steffen-karlsson — APPROVED** [review]
LGTM 💚🎉

**Pull request merged**